dcos_test_utils/etcd.py

```diff
@@ -0,0 +1,45 @@
+import os
+import subprocess
+
+from typing import List
+
+ETCDCTL_PATH = "/opt/mesosphere/bin/etcdctl"
+ETCD_ENDPOINTS = "127.0.0.1:2379"
+
+CA_CERT = "/run/dcos/pki/CA/ca-bundle.crt"
+
+
+def is_enterprise():
+    return os.getenv('DCOS_ENTERPRISE', 'false').lower() == 'true'
```

Comment:
Minor point, but the default argument could be `''` to avoid doing much work to check case and compare in that case.